blockscout-ens/bens-proto/proto/api_config_http.yaml

```diff
@@ -0,0 +1,18 @@
+type: google.api.Service
+config_version: 3
+
+http:
+  rules:
+    #################### DomainsExtractor ####################
+    - selector: blockscout.bens.v1.DomainsExtractor.SearchAddress
+      post: /api/v1/domains/search-address
```

Comment:
Lets also make it more in line with our other service apis: 
```suggestion
      post: /api/v1/domains/address:search
```

Reply:
doesn't it sound like you searching for address, not searching domains using address?

Reply:
maybe `/api/v1/{chain_id}/domains:reverse-search`